staging/2018/benchmarks/json/001_benchmarks_wi.json

```diff
@@ -153,21 +162,21 @@
     ]
   },
   {
-      "measureId": "204",
-      "benchmarkYear": 2016,
-      "performanceYear": 2018,
-      "submissionMethod": "cmsWebInterface",
-      "isToppedOut": false,
-      "deciles": [
-          0,
-          30.00,
-          30.00,
-          40.00,
-          50.00,
-          66.00,
-          70.00,
-          80.00,
-          90.00
-      ]
+    "measureId": "204",
+    "benchmarkYear": 2016,
+    "performanceYear": 2018,
+    "submissionMethod": "cmsWebInterface",
+    "isToppedOut": false,
+    "deciles": [
+      0,
+      30.00,
+      30.00,
+      40.00,
+      50.00,
+      60.00,
```

Comment:
This line is the only merge issue from #230

Reply:
Talked with Rocky, the is isToppedOut: false should not have been removed. Added it back in to keep things consistent.